lib/evmone/analysis.cpp

```diff
@@ -15,6 +15,18 @@ bool is_terminator(uint8_t c) noexcept
     return c == OP_JUMP || c == OP_JUMPI || c == OP_STOP || c == OP_RETURN || c == OP_REVERT ||
            c == OP_SELFDESTRUCT;
 }
+
+int load_1byte_immediate_value(const uint8_t* code, size_t code_size, size_t& index) noexcept
+{
+    return (index + 1 < code_size) ? code[++index] : 0;
```

Comment:
Does this mean that code not having an argument behaves the same as argument 0? And costs the same gas? Is this correct? (how is this with the code not having enough bytes after PUSH?)

Reply:
It's the same case as implicit zero bytes after any PUSH instructions.